domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -1754,7 +1754,7 @@ public void executeParamConnectionException(TestContext context) throws Exceptio
 
   @Test
   public void executeTrans(TestContext context) {
-    Async async1 = context.async();
+    Async async = context.async();
```

Comment:
You cannot reuse an Async. After async.complete() has been invoked once all other async.awaitSuccess will not wait but succeed immediately.
Please revert.